### PR TITLE
Samples for different azure function runtimes

### PR DESCRIPTION
**Pull request by geekwhocodes**
Added "samples" folder where users can add samples for different azure function runtimes. 

Added samples for :

- CSharp
  * Queue Output
  * Queue Trigger

- Python
  * Queue Output
  * Queue Trigger

### CODE REVIEW

**Comment by fabiocav** [comment]
Thank you for submitting this! We'll be reviewing this ASAP.

/cc @pragnagopa @jeffhollan

File: samples/Python/proxies.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+  "$schema": "http://json.schemastore.org/proxies",
+  "proxies": {}
```
Comment (pragnagopa) [inline samples/Python/proxies.json:3]: This sample is not adding any proxies. Please delete.

File: samples/Python/QueueTrigger/sample.dat [hunk 2]
```diff
@@ -0,0 +1 @@
+sample queue data
```
Comment (pragnagopa) [inline samples/Python/QueueTrigger/sample.dat:1]: this file is not needed unless you plan to include test scripts that use this file to invoke the function

File: samples/Python/QueueTrigger/readme.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+# RabbitMQ QueueTrigger - Python
```
Comment (pragnagopa) [inline samples/Python/QueueTrigger/readme.md:1]: Readme included in the repo and wiki are descriptive. Readme per sample is not needed

File: samples/Python/QueueOutput/sample.dat [hunk 4]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (pragnagopa) [inline samples/Python/QueueOutput/sample.dat:1]: delete this file

File: samples/Python/QueueOutput/readme.md [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# RabbitMQ QueueOutput - Python
+
```
Comment (pragnagopa) [inline samples/Python/QueueOutput/readme.md:2]: same here. Readme per sample is not needed

File: samples/CSharp/AzureFunctionsRabbitMQSamples/example.settings.json [hunk 6]
```diff
@@ -0,0 +1,8 @@
+{
+  "IsEncrypted": false,
+  "Values": {
+    "AzureWebJobsStorage": "UseDevelopmentStorage=true",
+    "FUNCTIONS_WORKER_RUNTIME": "dotnet",
```
Comment (pragnagopa) [inline samples/CSharp/AzureFunctionsRabbitMQSamples/example.settings.json:5]: Please create a folder 'Azure.Functions.Samples' under Samples folder and then sub directories per language:  `CSharp` , `Python` etc

File: samples/CSharp/AzureFunctionsRabbitMQSamples/OutputToQueue.cs [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Text;
+
+namespace AzureFunctionsRabbitMQSamples
+{
+    public class OutputToQueue
+    {
+        public OutputToQueue()
+        {
```
Comment (pragnagopa) [inline samples/CSharp/AzureFunctionsRabbitMQSamples/OutputToQueue.cs:17]: Please combine queue samples into one file

**Review by pragnagopa — CHANGES_REQUESTED** [review]
Thanks for the PR. Added few comments.

**Comment by geekwhocodes** [comment]
Thanks for the feedback, I will remove unnecessary files asap.

**Comment by geekwhocodes** [comment]
This is being redundant. I will add these examples in wiki itself.

**Pull request closed without merging**